package.json

```diff
@@ -59,29 +41,34 @@
     "window-or-global": "^1.0.1"
   },
   "devDependencies": {
-    "@storybook/addon-a11y": "^3.4.8",
-    "@storybook/addon-actions": "^3.4.8",
-    "@storybook/addon-knobs": "^3.4.8",
-    "@storybook/addon-links": "^3.4.8",
-    "@storybook/addons": "^3.4.8",
-    "@storybook/react": "^3.4.8",
+    "@svgr/cli": "^2.1.0",
     "babel-core": "^6.26.3",
-    "babel-eslint": "^8.2.5",
+    "babel-eslint": "^7.2.3",
+    "babel-jest": "^23.4.2",
+    "babel-polyfill": "^6.26.0",
     "babel-runtime": "^6.26.0",
     "big.js": "^5.1.2",
     "bundlesize": "^0.17.0",
     "ecstatic": "^3.2.1",
-    "eslint": "^5.1.0",
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-react-16": "^1.1.1",
+    "eslint": "^4.19.1",
     "eslint-plugin-react": "^7.10.0",
     "npm-run-all": "^4.1.3",
     "puppeteer": "^1.5.0",
+    "react-scripts": "1.1.4",
+    "react-test-renderer": "^16.4.1",
     "source-map-explorer": "^1.5.0",
     "standard": "^11.0.1"
   },
   "homepage": "./",
   "standard": {
     "parser": "babel-eslint"
   },
+  "browsers": [
+    ">0.25%",
+    "not dead"
```

Comment:
:laughing: 

Jokes aside, do we have a policy for this? So far we are quite creative:
- https://github.com/ipfs-shipyard/ipld-explorer-components/blob/cb745c55b9383b5cf394fa15508545b3f7fdc442/package.json#L80
- https://github.com/ipfs-shipyard/peer-pad/blob/e65fa08e6bce4627b58e6e82e0fdc0382a3b828f/config/createStyleRule.js#L28
- https://github.com/ipfs-shipyard/IGiS/blob/6a52b244ee52c294ac1a4c0e2042fae549b9e282/config/webpack.config.dev.js#L179 
- https://github.com/ipfs-shipyard/ipfs-share-files/blob/8217ffd4a4712e2175d4e3567a23108a8f58960d/config/webpack.config.prod.js#L226
- and we may have duplicate in this very repo: https://github.com/ipfs-shipyard/ipld-explorer/blob/ea4b8ccab65d964f2cc4fd57e4c93b21bfe6683d/config/webpack.config.prod.js#L226-L230

Reply:
We currently have no best practice for this. I was persuaded by https://jamie.build/last-2-versions that the old "last 2 versions" probably wasn't what we wanted.

"not dead" is now part of a recommended browser list, where `dead` is defined as

- `dead`: browsers from `last 2 version` query, but with less than 0.5%
  in global usage statistics and without official support or updates
  for 24 months. Right now it is `IE 10`, `IE_Mob 10`, `BlackBerry 10`,
  `BlackBerry 7`, and `OperaMobile 12.1`.

but the current browserlist defaults do look sensible

- * `defaults`: Browserslist’s default browsers
  (`> 0.5%, last 2 versions, Firefox ESR, not dead`).